try:
    print(x)
except Exception as e:
    print(e)


li = [323344,54,65,64545,6546,45]
print(li)